datasets/_global/eiti_soe_list/eiti_soe_list.yaml

```diff
@@ -0,0 +1,40 @@
+title: SOE List
+entry_point: crawler.py
+prefix: eiti-soe
+coverage:
+  frequency: daily
+  start: "2024-08-09"
+deploy:
+  schedule: "@weekly"
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  List of all SOEs operating in EITI implementing countries.
+description: |
+  This dataset covers SOEs’ payments to government and other disclosures related to oil, 
+  gas and mining activities in EITI implementing countries.
```

Comment:
This dataset doesn't really cover the payments - it only really lists the entities.

You could expand the dataset description a bit with something like "this is made available to facilitate identification of channels for corruption."